ackhandler/sent_packet_handler.go

```diff
@@ -386,6 +390,18 @@ func (h *sentPacketHandler) SendingAllowed() bool {
 	return !maxTrackedLimited && (!congestionLimited || haveRetransmissions)
 }
 
+func (h *sentPacketHandler) TimeUntilSend() time.Time {
+	return h.nextPacketSendTime
+}
+
+func (h *sentPacketHandler) ShouldSendNumPackets() int {
+	delay := h.congestion.TimeUntilSend(h.bytesInFlight)
+	if delay == 0 || delay > protocol.MinPacingDelay {
+		return 1
+	}
+	return int(math.Ceil(float64(protocol.MinPacingDelay) / float64(delay)))
```

Comment:
Nit: Why Ceil() here? If the float is 1.000000001, don't we want it to come out as 1?

Reply:
Because otherwise we'd set deadlines shorter than `protocol.MinPacingDeadline`. To be precise, if we round that value, we might set deadlines as short as `protocol.MinPacingDeadline / 1.5`.